OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -120,6 +121,19 @@ void Muscle::updateFromXMLNode(SimTK::Xml::Element& aNode, int versionNumber)
                 }
             }
         }
+        if (versionNumber < 30516) {
+            // Find GeometryPath node and insert <default_color>
+            SimTK::Xml::element_iterator  geomPathIter = aNode.element_begin("GeometryPath");
+            if (geomPathIter != aNode.element_end()) {
+                SimTK::Xml::element_iterator  defaultColorIter = geomPathIter->element_begin("default_color");
+                if (defaultColorIter == geomPathIter->element_end()) {
+                    SimTK::Xml::Element myDefaultColorEement("default_color");
+                    myDefaultColorEement.setValue(".8 .1 .1"); // DefaultMuscleColor
+                    geomPathIter->appendNode(myDefaultColorEement);
+                }
+            }
+        }
```

Comment:
Thank you for explaining. My understanding is that all new model files will now have the color show up explicitly in the XML file (for muscles). I find that a little unfortunate, but fine for now.